troubadix/standalone_plugins/no_solution.py

```diff
@@ -275,7 +275,11 @@ def main():
 
     summary = check_no_solutions(files, milestones, arguments.snooze)
 
-    print_report(term, summary, arguments.threshold, root)
+    found_vts = sum(len(entries) for _, entries in summary)
+
+    print_report(term, summary, arguments.threshold, root, found_vts)
+
+    return 1 if len(found_vts) > 0 else 0
```

Comment:
```suggestion
    sys.exit(1 if found_vts > 0 else 0)
```
Can't take `len` of an int and the return code is ignore for me, because `main()` is called directly by poetry (and AFAIK also without it when installed as usual)

Reply:
Yeah, VS code didnt work like that apparently. Fixed